### PR TITLE
T10 f22 61 profile  implement delete profile on node

### PR DESCRIPTION
**Pull request by Zevez7**
Postman link to test delete profile

https://api.postman.com/collections/8015820-0fd3148f-5c0f-4a0f-ab92-1a6ea0d476cd?access_key=PMAT-01GJYJ12MPKP3SGJEDGZVEPJPB

### CODE REVIEW

**Review by hzhou74 — APPROVED** [review]
postman checked. no problem

File: daos/ProfileDao.ts [hunk 1]
```diff
@@ -21,19 +21,52 @@ export default class ProfileDao implements ProfileDaoI {
...
+   * @param req Request for the API endpoints
+   * @param res Response from the node server
+   * @returns all profile
+   */
   async findAllProfile(): Promise<Profile[]> {
     return await ProfileModel.find();
   }
 
+  /**
+   * find one profile by user id controller
```
Comment (ujjvalksharma) [inline daos/ProfileDao.ts]: PLease change js doc.

File: interfaces/ProfileDao.ts [hunk 2]
```diff
@@ -26,4 +26,10 @@ export default interface ProfileDao {
    * @param uid user id
    */
   updateProfileByUserId(uid: string, profile: Profile): Promise<any>;
+
+  /**
+   * Delete Profile by user id
+   * @param uid user id
+   */
```
Comment (ujjvalksharma) [inline interfaces/ProfileDao.ts:34]: Please add return type.

**Review by ujjvalksharma — CHANGES_REQUESTED** [review]
Please apply the changes and make the PR again

File: interfaces/ProfileDao.ts (hunk 2 above)
Comment (Zevez7) [reply interfaces/ProfileDao.ts:34]: I don't think you add return type to interface. According to Dr. Jose code there's no return type for interface

File: daos/ProfileDao.ts (hunk 1 above)
Comment (Zevez7) [reply daos/ProfileDao.ts]: changed

**Pull request merged**